packages/static-hosting/lib/static-hosting.ts

```diff
@@ -575,13 +605,65 @@ export class StaticHosting extends Construct {
       );
     }
 
+    // Create CORS behavior config if corsAllowOrigins is specified
+    const corsBehavior: ResponseHeadersCorsBehavior | undefined =
+      props.corsAllowOrigins && props.corsAllowOrigins.length > 0
+        ? {
+            accessControlAllowCredentials: false,
+            accessControlAllowHeaders: ["*"],
```

Comment:
this whole thing should be customizable IMO, rather than `props.corsAllowOrigins` you could have a `Partial<ResponseHeadersCorsBehavior>` or similar and use that. You could still have sensible defaults using this method